src/upscaler.py

```diff
@@ -703,7 +706,17 @@ def run(self):
 
                 # downscale frames with Lanczos
                 Avalon.info(_('Lanczos downscaling frames'))
-                shutil.rmtree(self.extracted_frames)
+                try:
+                    shutil.rmtree(self.extracted_frames)
+                except:
+                    if platform.system() == 'Windows':
+                        Avalon.info(_("Change file's attribute."))
```

Comment:
This line is not clear. What file's what attribute are you changing?
Also, there should be no periods at the end of command line outputs.